registry.json

```diff
@@ -4279,18 +4279,18 @@
     "github": "grahamde8313"
   },
   {
-    "ticker": "LOVLK",
```

Comment:
This PR is removing LOVLK... You need to add your pool only. Ping me on CSPA Telegram if you are having trouble